scripts/run-e2e-tests.sh

```diff
@@ -30,16 +30,29 @@ source "$SCRIPTS_DIR/pytest-image-runner.sh"
 
 ensure_cluster() {
     local cluster_create="$(get_cluster_create)"
-    if [[ "$cluster_create" == true ]]; then
-        local cluster_name=$(_get_kind_cluster_name)
+
+    if [[ "$cluster_create" != true ]]; then
+        info_msg "Testing connection to existing cluster ..."
+        _ensure_existing_context
+    else
+        local cluster_name=$(get_cluster_name)
+        local controller_install=${2:-true}
```

Comment:
Could you please move all of the positional arguments into local variables at the top of this method, so that it's clear what the inputs are. Then you can copy them into their respective variables further down in the method as needed.

Reply:
moved [4c321c2](https://github.com/aws-controllers-k8s/test-infra/pull/338/commits/4c321c266b567a611a20ab7fa07da60b49e85785)